include/ui/hexedit.h

```diff
@@ -105,6 +105,7 @@ class HexEdit : public QAbstractScrollArea {
   /** Number of rows displayed on the screen (calculated from window and font
    * height) */
   qint64 rowsOnScreen_;
+  qint64 totallyVisibleRows_;
```

Comment:
Should be `totally_visible_rows_`, unfortunately the old code doesn't use Google style (I'll refactor it someday).